### PR TITLE
Add #zougla community under News & Politics

### PR DESCRIPTION
**Pull request by duckspike**

### CODE REVIEW

File: content/community/zougla.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+aliases: [ 'zougla' ]
+title: "#zougla"
+date: 2019-04-11T16:19:28Z
+draft: false
+type: community
+kind: "news_and_politics"
+---
+
+Καλωσήρθατε στο #zougla. Για να συνδεθείτε στο chat, μπείτε στη σελίδα https://chat.zougla.gr/webchat(new tab) και στην συνέχεια πατήστε: Start
```
Comment (linuxdaemon) [inline content/community/zougla.md]: I believe this is supposed to be a link

File: content/community/zougla.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Δεν είναι απαραίτητο να ορίσετε Nick. Κάθε φορά που συνδέεστε, αυτόματα ορίζεται ένα τυχαίο nick με το οποίο μπορείτε να συνδεθείτε.
+
+Κανόνες και πολιτική λειτουργίας: 
+
+Το zougla.gr(href blank zougla.gr) κατατάσσεται στις καλύτερες ιστοσελίδες ενημέρωσης στην Ελλάδα, σύμφωνα με το Alexa Rank(new tab https://www.alexa.com/siteinfo/zougla.gr).
```
Comment (linuxdaemon) [inline content/community/zougla.md]: Here too

File: content/community/zougla.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+Κύριος στόχος του καναλιού είναι η ανταλλαγή απόψεων και η δημιουργία συζητήσεων που δεν προσβάλλουν την ανθρώπινη υπόσταση και δεν στοχοποιούν συγκεκριμένες ομάδες ατόμων.
+
+Βασικός στόχος είναι η ψυχαγωγία και όχι η προώθηση πολιτικών και θρησκευτικών πεποιθήσεων, απόψεις που δεν ενστερνίζονται όλοι. 
+
+Απαγορεύεται η χρήση προσωνύμιων που έχουν ρατσιστικό/σεξουαλικό ή άλλο χαρακτήρα, που θα μπορούσε να προσβάλλει κάποιο άλλο μέλος ή ολόκληρο το κανάλι, κάθε είδους διαφημίσεις Spam/Ιστοσελίδων/Blog που έχουν ως σκοπό οποιονδήποτε κερδοσκοπικό χαρακτήρα και τέλος κάθε είδους μορφή κατάχρησης όπως Flooding/Abuse των υπηρεσιών που σας παρέχονται.
+
+Οι διαχειριστές/operators του #zougla δεν είναι σε θέση να γνωρίζουν/παρακολουθούν τα προσωπικά σας μηνύματα και επομένως δεν φέρουν καμία ευθύνη για ενδεχόμενες μη νόμιμες ενέργειες σας. Σε περίπτωση που διαπιστώσετε κάποια παράνομη ενέργεια, μπορείτε να ενημερώσετε τους operators στο κανάλι #zougla-opers(new tab https://webchat.snoonet.org/#zougla-opers) ή να στείλετε email στην φόρμα επικοινωνίας(new tab https://chat.zougla.gr/contact).
```
Comment (linuxdaemon) [inline content/community/zougla.md]: And here

File: content/community/zougla.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+Σύχνες ερωτήσεις:
+
+Ποια είναι τα πρώτα βήματα που πρέπει να ακολουθήσω για να προσαρμοστώ καλύτερα στο chat;
+
+a) Όλοι οι χρήστες που συνδέονται στο #zougla(new tab https://chat.zougla.gr/webchat) οφείλουν να διαβάσουν τους κανόνες και την πολιτική λειτουργίας για να γνωρίζουν τι ΔΕΝ πρέπει να κάνουν κατά την είσοδο τους.
```
Comment (linuxdaemon) [inline content/community/zougla.md]: Here

File: content/community/zougla.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+Οι πραγματικές IP διευθύνσεις των χρηστών που συνδέονται στο #zougla δεν είναι ορατές για λόγους προστασίας.   
+
+Ο κωδικός πρόσβασης που έχω βάλει είναι ασφαλής;
+
+Οι κωδικοί πρόσβασης είναι κρυπτογραφημένοι και δεν μπορεί να τους γνωρίζει κάποιος άλλος χρήστης/διαχειριστής. Σε περίπτωση που ξεχάσετε τον κωδικό σας, μπορείτε να κάνετε ανάκτηση μέσω email. Για περισσότερες πληροφορίες σχετικά με την ανάκτηση του λογαριασμού σας μπορείτε να απευθυνθείτε στο κανάλι #zougla-opers
+
+Θέλω να γίνω Operator/Moderator(@) στο #Zougla, τι πρέπει να κάνω;
+
+Θα πρέπει να κάνετε αίτηση γι αυτό στη σελίδα https://chat.zougla.gr/request-oper new tab
```
Comment (linuxdaemon) [inline content/community/zougla.md]: here

File: content/community/zougla.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+Δεν είναι απαραίτητο να γράφετε με Ελληνικούς χαρακτήρες. Για κάποιους η χρήση των Greeklish ενδεχομένως να είναι πιο γρήγορη. Πάραυτα, καλό θα ήταν να γράφουμε όλοι με Ελληνικούς χαρακτήρες διότι έτσι γίνεται πιο εύκολη η ανάγνωση.
+
+Μπορώ να χρησιμοποιήσω μικρόφωνο για να μιλάω με άλλα άτομα;
+
+Η απάντηση είναι ναι, γίνεται να μιλήσετε με μικρόφωνο. Προϋπόθεση για να γίνει αυτό είναι και οι δύο συνομιλητές να έχουν συνδεθεί από το https://webchat.snoonet.org/#zougla και να επιλέξουν το εικονίδιο της κλήσης. Μπορεί να χρησιμοποιηθεί είτε στο κεντρικό κανάλι, είτε σε προσωπικό μήνυμα.
+
+Θέλω να επικοινωνήσω με κάποιον από την ομάδα του #zougla, που μπορώ να απευθυνθώ;
+
+Για οποιεσδήποτε ερωτήσεις/καταγγελίες που αφορούν ΜΟΝΟ το #zougla μπορείτε να απευθυνθείτε στο κανάλι #zougla-opers(new tab https://webchat.snoonet.org/#zougla-opers) ή να στείλετε email μέσω της φόρμας επικοινωνίας.(new tab https://chat.zougla.gr/contact)
```
Comment (linuxdaemon) [inline content/community/zougla.md]: and here

**Review by linuxdaemon — CHANGES_REQUESTED** [review]

File: content/community/zougla.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+kind: "news_and_politics"
+---
+
+Καλωσήρθατε στο #zougla. Για να συνδεθείτε στο chat, μπείτε στη σελίδα [https://chat.zougla.gr/webchat](https://chat.zougla.gr/webchat) και στην συνέχεια πατήστε: Start
+
+Δεν είναι απαραίτητο να ορίσετε Nick. Κάθε φορά που συνδέεστε, αυτόματα ορίζεται ένα τυχαίο nick με το οποίο μπορείτε να συνδεθείτε.
+
+Κανόνες και πολιτική λειτουργίας: 
+
+Το [zougla.gr](http://zougla.gr/) κατατάσσεται στις καλύτερες ιστοσελίδες ενημέρωσης στην Ελλάδα, σύμφωνα με το [Alexa Rank](https://www.alexa.com/siteinfo/zougla.gr)
```
Comment (linuxdaemon) [inline content/community/zougla.md]:
```suggestion
Το [zougla.gr](https://zougla.gr/) κατατάσσεται στις καλύτερες ιστοσελίδες ενημέρωσης στην Ελλάδα, σύμφωνα με το [Alexa Rank](https://www.alexa.com/siteinfo/zougla.gr)
```

File: content/community/zougla.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+Μπορώ να χρησιμοποιήσω μικρόφωνο για να μιλάω με άλλα άτομα;
+
+Η απάντηση είναι ναι, γίνεται να μιλήσετε με μικρόφωνο. Προϋπόθεση για να γίνει αυτό είναι και οι δύο συνομιλητές να έχουν συνδεθεί από το [https://webchat.snoonet.org/#zougla](https://webchat.snoonet.org/#zougla) και να επιλέξουν το εικονίδιο της κλήσης. Μπορεί να χρησιμοποιηθεί είτε στο κεντρικό κανάλι, είτε σε προσωπικό μήνυμα.
+
+Θέλω να επικοινωνήσω με κάποιον από την ομάδα του #zougla, που μπορώ να απευθυνθώ;
+
+Για οποιεσδήποτε ερωτήσεις/καταγγελίες που αφορούν ΜΟΝΟ το #zougla μπορείτε να απευθυνθείτε στο κανάλι [#zougla-opers](https://webchat.snoonet.org/#zougla-opers) ή να στείλετε email μέσω της φόρμας [επικοινωνίας](https://chat.zougla.gr/contact).
+
+Μην στέλνετε email για θέματα που αφορούν την κεντρική ιστοσελίδα του [zougla.gr](http://zougla.gr)
```
Comment (linuxdaemon) [inline content/community/zougla.md]:
```suggestion
Μην στέλνετε email για θέματα που αφορούν την κεντρική ιστοσελίδα του [zougla.gr](https://zougla.gr)
```

**Review by linuxdaemon — CHANGES_REQUESTED** [review]

**Review by linuxdaemon — APPROVED** [review]

**Comment by noeatnosleep** [comment]
Somethingsomething GDPR related. Here's the new text they've requested: 

https://acn.gr/d/zougla-texts

**Pull request merged**